### PR TITLE
add support for capturing headers

### PR DESCRIPTION
**Pull request by bioball**
For #7

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const path = require("path");
 const _ = require("lodash");
 const moment = require("moment");
 const cli = require('cli').enable('catchall').enable('status');
-
+const MailParser = require('mailparser').MailParser;
```
Comment (jcdelmas) [inline index.js]: It seems that this import is not used.

**Comment by bioball** [comment]
Hey @jcdelmas, that line actually isn't part of this PR. It was deleted in commit 6ce02bf.

**Comment by jcdelmas** [comment]
Yes, sorry !

**Comment by bioball** [comment]
Thanks! Can you publish a new version of this module? :D

**Pull request merged**